Cryptlib/Pk/CryptPkcs7VerifyCommon.c

```diff
@@ -814,6 +815,8 @@ Pkcs7Verify (
   CONST UINT8 *Temp;
   UINTN       SignedDataSize;
   BOOLEAN     Wrapped;
+  CONST CHAR8 *RequiredEku[1];

```

Comment:
Should EKU be a parameter to this function instead, along with a size (for multiple EKUs required), or push the EKU verification up a level ?  The parameter path is probably cleaner, but deviates from the API surface used in UEFI BaseCrypt (where these are separate functions).  Cleanest might be to add a Pkcs7VerifyWithEku() that calls Pkcs7Verify() and VerifyEKUsInPkcs7Signature(), and the supplied EKU list can optionally be NULL (but probably won't be in your case).  Perhaps reference https://github.com/microsoft/mu_plus/blob/release/202008/MfciPkg/Private/Library/MfciPolicyParsingLib/MfciPolicyParsingLib.c function ValidateSignature() (which limits to 1 EKU currently).